build/cni/Makefile

```diff
@@ -18,7 +18,7 @@
 #	[ARCH=amd64] [CNI_RELEASE=v0.3.0] make (build|push)
 
 ARCH?=amd64
-CNI_RELEASE?=07a8a28637e97b22eb8dfe710eeae1344f69d16e
+CNI_RELEASE?=0799f5732f2a11b329d9e3d51b9c8f2e3759f2ff
```

Comment:
I'm not sure why we don't just use the tag, but I guess we can think about that at a later date.